src/electron.ts

```diff
@@ -53,26 +55,20 @@ app.on('ready', () => {
   });
 
   ipcMain.on('close', () => {
-    if (current) {
-      mainWindowState.saveState(current);
-      unregisterShortcuts();
-      current.close();
-      current = null;
-    }
+    handleWindowsOnClose();
   });
 
   ipcMain.on('closeApp', () => {
-    unregisterShortcuts();
-    if (current) {
-      mainWindowState.saveState(current);
-      current.close();
-      current = null;
-    }
+    handleWindowsOnClose();
   });
 });
 
 app.on('browser-window-created', (e: Event, win: Electron.BrowserWindow) => {
   current = win;
+  windows = new Array<Electron.BrowserWindow>()
```

Comment:
The idea is to initiate array since it is declared as null, but I will apply comment below. Thanks.